HTML/HTML.sublime-syntax

```diff
@@ -237,8 +237,18 @@ contexts:
     - match: <>
       scope: invalid.illegal.incomplete.html
   entities-common:
-    - match: (&)([a-zA-Z0-9]+|#[0-9]+|#x[0-9a-fA-F]+)(;)
-      scope: constant.character.entity.html
+    - match: (?i)(&#x)(\h+)(;)
+      scope: constant.character.entity.hexadecimal.html
+      captures:
+        1: punctuation.definition.entity.html
+        3: punctuation.definition.entity.html
+    - match: (?i)(&#)([0-9]+)(;)
+      scope: constant.character.entity.decimal.html
+      captures:
+        1: punctuation.definition.entity.html
+        3: punctuation.definition.entity.html
+    - match: (?i)(&)([a-zA-Z0-9]+)(;)
```

Comment:
If this is case-insensitive, why put both upper and lower A-Z?

Reply:
I went back and forth on this a couple of times, and I decided to strip the `(?i)` flags. The new expressions are a bit more explicit.